spec/apps/app-names/app-names.spec.js

```diff
@@ -6,7 +6,7 @@ export default function () {
       singleSpa.registerApplication('./app-names.app.js', () => System.import('./app-names.app.js'), location => location.hash === '#app-names')
     })
 
-    it(`should return all registered app names up to this point regardless of activity`, (done) => {
```

Comment:
I think I never correctly implemented this test. I'm embarrassed that I missed that the test was never running